marko.json

```diff
@@ -21,6 +21,7 @@
         "@html-attributes": "expression",
         "@class": "string",
         "@disabled": "boolean",
+        "@type": "string",
```

Comment:
Move this below `class`, since it kind of belongs with the html attributes.

Reply:
https://github.com/cestdiego/ebayui-core/commit/3aa5f90